ch.elexis.core.data/src/ch/elexis/core/data/util/DBUpdate.java

```diff
@@ -431,7 +432,11 @@
 			// 3.4.1
 			FILE_LOCATED,
 			// 3.4.2
-			FILE_LOCATED
+			FILE_LOCATED,
+			// 3.4.3
+			"UPDATE user_ SET `id` = LOWER(`id`);"+
+			ALTER_TABLE+Konsultation.TABLENAME+ADD+Konsultation.FLD_TIME+" CHAR(6);"
```

Comment:
I object about addding a CHAR field with time info.
Instead I suggest adding support for proper handling a SQL datetime column (ISO/IEC 9075-11:2)
See 
* https://dev.mysql.com/doc/refman/5.7/en/date-and-time-literals.html
* https://www.postgresql.org/docs/9.1/static/datatype-datetime.html
* (H2) only has timestamp: http://h2database.com/html/datatypes.html#date_type

The standards are availble (for a too big for me price) under https://www.iso.org/search/x/query/SQL.